dbt_platform_helper/templates/pipelines/codebase/overrides/types.ts

```diff
@@ -26,6 +26,7 @@ export interface PipelinesConfiguration {
     codebase_pipelines: Array<{
         name: string;
         repository: string;
+        deploy_repo_branch?: string;
```

Comment:
Can we stick with the `repository` word for consistency.
```suggestion
        deploy_repository_branch?: string;
```